libraries/botframework-streaming/src/namedPipe/namedPipeServer.ts

```diff
@@ -40,6 +40,10 @@ export class NamedPipeServer implements IStreamingTransportServer {
      * @param autoReconnect Optional setting to determine if the client sould attempt to reconnect automatically on disconnection events. Defaults to true.
      */
     public constructor(baseName: string, requestHandler?: RequestHandler, autoReconnect: boolean = true) {
+        if (!baseName) {
+            throw new TypeError('NamedPipeServer: Missing baseName parameter');

```

Comment:
Throwing `TypeError` here for missing parameter, but you throw an `Error` here for the same thing: https://github.com/microsoft/botbuilder-js/pull/1627/files#diff-f90ca2f557a7f5ac0cad4a842c08d731R37

Reply:
Yes, the difference here is because the line you highlighted is already throwing an `Error`.

On this [line](https://github.com/microsoft/botbuilder-js/pull/1627/files#diff-f90ca2f557a7f5ac0cad4a842c08d731R37) I would rather throw a TypeError, but I've avoided changing the type here and plan on doing it in master for the R8 release.